dusty/config.py

```diff
@@ -111,8 +111,9 @@ def _load_ssh_auth_pre_yosemite():
 def _set_ssh_auth_sock(ssh_auth_sock):
     daemon_warnings.clear_namespace('ssh')
     if ssh_auth_sock:
-        logging.info("Setting SSH_AUTH_SOCK to {}".format(ssh_auth_sock))
-        os.environ['SSH_AUTH_SOCK'] = ssh_auth_sock
+        if ssh_auth_sock != os.getenv('SSH_AUTH_SOCK'):
+            logging.info("Setting SSH_AUTH_SOCK to {}".format(ssh_auth_sock))
+            os.environ['SSH_AUTH_SOCK'] = ssh_auth_sock
```

Comment:
how is this code any different than before?   Before you set it every time, now you are only setting it if they are different.   But if they are the same, setting the value does not really change anything?